### PR TITLE
Update codebase to python 3

### PR DESCRIPTION
**Pull request by alexbarrera**
This `dev` branch addresses a couple things needed moving forward:
- Migrate code to Python 3 (worth noting that the new code will not be compatible with Python 2.x anymore)
- Add `setup.py` and some other necessary file for packaging the tool and make it installable with `pip install /path/to/ggr-cwl-ipynb-gen`

Since we don't have unittests or CI yet, it might be one for one of the reviewers to download and test before merging (if you haven't yet)

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -0,0 +1,9 @@
+jinja2 >=2.8
+nbformat >=4.0.1
+numpy >=1.10.4
+pandas >=0.17.1
+xlrd >=1.0.0
+ruamel.yaml >=0.11.11
+
+setuptools
+pymongo
```
Comment (tncowart) [inline requirements.txt:9]: Shouldn't pymongo be versioned too?

File: requirements.txt [hunk 2]
```diff
@@ -0,0 +1,9 @@
+jinja2 >=2.8
+nbformat >=4.0.1
+numpy >=1.10.4
+pandas >=0.17.1
+xlrd >=1.0.0
+ruamel.yaml >=0.11.11
+
+setuptools
```
Comment (tncowart) [inline requirements.txt:8]: I don't think you need setup tools here because it's mentioned in pyproject.toml

File: MANIFEST.in [hunk 3]
```diff
@@ -0,0 +1 @@
+include templates/*
```
Comment (tncowart) [inline MANIFEST.in:1]: I don't think you need a manifest file because you use `data_files` in setup.py

**Review by tncowart — CHANGES_REQUESTED** [review]
In addition to my comments I don't think you should check in the symlinks.

File: requirements.txt (hunk 1 above)
Comment (alexbarrera) [reply requirements.txt]: I didn't versioned pymongo because I'm not sure where to draw the line of the oldest version that should be accepted

**Comment by alexbarrera** [comment]
What's the argument to remove the `symlinks`? I wanted to keep them for backwards compatibility, although admittedly this should be work with Python 2.x, so perhaps I shouldn't worry... Curious about the justification to remove them though

File: requirements.txt (hunk 1 above)
Comment (tncowart) [reply requirements.txt]: Your initial commit was 4/26/2017 and it looks like the newest version of pymongo at the time was 3.4.0, so probably that version (it's only at 3.11 now, 4 years later).

**Comment by tncowart** [comment]
Just to declutter the repo. I don't feel super strongly about it though.

**Comment by alexbarrera** [comment]
Got it, yeah, I imagined that was it. I also feel like symlinks are prob something that can stay away from the repo, not common to find them committed.

Thanks for all the comments!!

**Review by tncowart — APPROVED** [review]

**Comment by tncowart** [comment]
Closes #11

**Pull request merged**